modules/ROOT/pages/bootloader-updates.adoc

```diff
@@ -0,0 +1,73 @@
+= Updating the bootloader
+
+== bootupd
```

Comment:
Should we mention that this is just for EFI for now?

Reply:
Done.